lib/linguist/languages.yml

```
@@ -6784,6 +6784,21 @@ V:
   codemirror_mode: go
   codemirror_mime_type: text/x-go
   language_id: 603371597
+VB6:
```

I think we should spell this out as it appears to be commonly referred to in full from a quick search.

```suggestion
Visual Basic 6.0:
```

---

I don't have a strong opinion about this personally and I can indeed see that the spelled out version is indeed used often.
I've made the change accordingly. https://github.com/github/linguist/pull/6124/commits/d443d5283bbdb8ca37a14a38ad1203437f3f240f